week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,32 +1,40 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + " and I am " + age + " years old";
+}
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+  remainder = a % b;
```

Comment:
The variable remainder has no declaration keyword, meaning it's a global variable 
Search for (declaring a variable without var)
```const remainder = a % b;```